panel/widgets/tables.py

```diff
@@ -1849,7 +1849,6 @@ def _config_columns(self, column_objs: List[TableColumn]) -> List[Dict[str, Any]
                 col_dict['editor'] = 'list'
                 if col_dict.get('editorParams', {}).get('values', False) is True:
                     del col_dict['editorParams']['values']
-                    col_dict['editorParams']['valuesLookup']
```

Comment:
@maximlt any ideas? Was changed here: https://github.com/holoviz/panel/commit/d2e6671e5d85676b2517aa1fdfe4dd870f9e8cdc

Seems like at best this behaves like an assert.